ARCs/arc-draft_alternative_0014.md

```diff
@@ -0,0 +1,285 @@
+---
+arc: 14
+title: Algorand Standard for Authentication
+description: Use Algorand accounts to authenticate with third-party services
+author: Stefano De Angelis <stefano@algorand.com>
+discussions-to: https://github.com/algorandfoundation/ARCs/pull/41
+status: Draft
+type: Meta
+created: 2022-04-05
+---
+
+# Algorand Standard for Authentication
+
+> This ARC is intended to be an alternative to PR #41 [Create arc-0014](https://github.com/algorandfoundation/ARCs/pull/41).
```

Comment:
You can remove this part as the other PR is now closed

Reply:
Done.